src/main/java/model/input/file/DataFile.java

```diff
@@ -251,4 +302,74 @@ public void setFirstRowAsHeader(boolean firstRowAsHeader) {
 	public String toString() {
 		return "[" + path + "]";
 	}
+
+	/**
+	 * Sets the standard metadata value.
+	 *
+	 * @param name The name that the metadata column will get
+	 * @param type The type of the column
+	 */
+	public void createMetaDataValue(String name, String type) throws ClassNotFoundException {
```

Comment:
Kijk eens naar de documentatie voor ClassNotFoundException en bedenk dan nog eens of hij wel de goeie is.